lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.scss

```diff
@@ -84,18 +84,8 @@ $panel-properties-height: 56px !default;
         }
 
         &-tags {
-            adf-tags-creator {
-                .adf-tags-creation {
-                    padding-right: 0;
-                }
-
-                &.adf-creator-with-existing-tags-panel {
-                    background: var(--adf-theme-background-dialog-color);
-                }
-            }
-
-            [hidden] {
-                visibility: hidden;
+            #{$mat-chip} {
```

Comment:
Is it necessary to use this selector here? Can we have a class or id that we are adding directly in the html/ts? The plan is to eventually get rid of the mat-selectors.scss file, so this will have to be changed at some point.

Reply:
yeah, it was possible :)